helmfile/charts/networkpolicy-service/templates/dex/allow-dex.yaml

```diff
@@ -0,0 +1,62 @@
+{{ if .Values.dex.enabled }}
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: allow-dex
+  namespace: dex
+spec:
+  policyTypes:
+    - Ingress
+    - Egress
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/name: dex
+  ingress:
```

Comment:
I think there are more things talking to dex in the cluster, at least grafana: https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/values/kube-prometheus-stack-sc.yaml.gotmpl#L261

Reply:
Added the missing dex connections